zeebe/broker/src/main/java/io/camunda/zeebe/broker/exporter/repo/ExporterRepository.java

```diff
@@ -97,16 +97,19 @@ public ExporterDescriptor load(final String id, final ExporterCfg config)
   private void validate(final ExporterDescriptor descriptor) throws ExporterLoadException {
     try {
       final Exporter instance = descriptor.newInstance();
-      final ExporterContext context =
+      try (final var context =
           new ExporterContext(
               LOG,
               descriptor.getConfiguration(),
               NULL_PARTITION_ID,
               new SimpleMeterRegistry(),
-              InstantSource.system());
+              InstantSource.system())) {
 
-      ThreadContextUtil.runCheckedWithClassLoader(
-          () -> instance.configure(context), instance.getClass().getClassLoader());
+        ThreadContextUtil.runCheckedWithClassLoader(
+            () -> instance.configure(context), instance.getClass().getClassLoader());
+      } finally {
+        instance.close();
+      }
```

Comment:
:thought_balloon: Huh, interesting that we didn't close the instance before. How did you notice this?

Reply:
I noticed because I made the ExporterContext Autocloseable and surrounded its constructor with try-with-resources.
But I think in this we don't need to close it explicitly as the Exporter instance itself will close the context. 
So I will leave only the `instance.close()`